utils.go

```diff
@@ -512,6 +512,20 @@ func isAmzHeader(headerKey string) bool {
 	return strings.HasPrefix(key, "x-amz-meta-") || strings.HasPrefix(key, "x-amz-grant-") || key == "x-amz-acl" || isSSEHeader(headerKey) || strings.HasPrefix(key, "x-amz-checksum-")
 }
 
+var supportedInternalHeaders = map[string]bool{
+	"x-minio-internal-server-side-encryption-sealed-key":     true,
+	"x-minio-internal-inline-data":                           true,
+	"x-minio-internal-server-side-encryption-seal-algorithm": true,
+	"x-minio-internal-server-side-encryption-iv":             true,
+	// Add more supported headers here.
+	// Must be lower case.
+}
```

Comment:
We shouldn't mention it in this fashion, instead it must be `x-minio-replication-server-side-encryption-sealed-key` follow this for others as well. 

Convert this on the target to relevant internal values.

Reply:
ok, will do the changes